index.html

```diff
@@ -10,7 +10,7 @@ <h1 class="offset-top" id="introduction">
       <small class="ml-2 text-muted">v{{- api.info.version -}}</small>
     </h1>
     {%- if api.info.description -%}
-      <p class="lead">{{- api.info.description -}}</p>
+      <p class="lead">{{- api.info.description | markdownify -}}</p>
```

Comment:
Change `<p>` to `<div>`

Reply:
[Done](https://github.com/robertlove/jekyll-openapi/compare/ef3938c60162179b98d344d7d399305503da7d0f..39678c89071c2531c37b76b5730126c8a2d02236).